.gitignore

```diff
@@ -1,5 +1,7 @@
 *.o
 src/*.o-*
+src-i386/
+src-x64/
```

Comment:
On Windows, x64 building the package generates these folders with binary output. They're not needed in the version control, so I've added them to .gitignore. If you'd prefer to keep them out of .gitignore, I could of course just ignore them manually!